scripts/ci/breaking_change_test.py

```diff
@@ -119,7 +121,18 @@ def get_pipeline_result():
             "Status": "Succeeded",
             "Content": ""
         })
-    print(json.dumps(pipeline_result, indent=4))
+
+    result_length = len(json.dumps(pipeline_result, indent=4))
```

Comment:
I think the content size be reduced a lot without indent=4

Reply:
The bot will add extra characters, so the number of characters cannot be accurately calculated. Adding indent=4 is just a rough estimate.
```
<!-- Azure CLI Extensions Breaking Change Test --><details>
<summary>️✔️Azure CLI Extensions Breaking Change Test</summary>
 
><details>
> <summary>️✔️Non Breaking Changes</summary>
>
>
></details>
</details>
```